Apps/AdminWebClient/src/Server/Services/CsvExportService.cs

```diff
@@ -83,6 +122,44 @@ public Stream GetRatings(DateTime? startDate, DateTime? endDate)
             return GetStream<Rating, UserProfileCsvMap>(profiles.Payload);
         }
 
+        /// <inheritdoc />
+        public async Task<Stream> GetInactiveUsers(int inactiveDays)
+        {
+            this.logger.LogDebug("Getting inactive users since {InactiveDays} day(s) from last login....", inactiveDays);
+
+            List<AdminUserProfileView> inactiveUsers = new List<AdminUserProfileView>();
+
+            // Inactive admin user profiles from DB
+            DBResult<IEnumerable<AdminUserProfile>> profilesResult = this.adminUserProfileDelegate.GetInactiveAdminUserProfiles(inactiveDays);
+
+            if (profilesResult.Status == DBStatusCode.Read && profilesResult.Payload.ToList().Count > 0)
```

Comment:
Nit: `profilesResult.Payload.ToList().Count` can be replaced with `profilesResult.Payload.Count()` which should avoid creating an extra List object.

Reply:
Addressed. Used *.Any()